src/bubbleSpec.ts

```diff
@@ -16,6 +16,10 @@ export interface BubbleSpecPattern {
   borderStyle?: string; // not implemented or fully designed yet
   backgroundColors?: string[]; //  just 1 color for solid, multiple for gradient. Omit for white. Top to bottom.
```

Comment:
Let's add a comment about what the format of the background color should look like, now that we have it.